defaults/main.yml

```diff
@@ -164,6 +163,13 @@ mariadb_sst_user:
     password: "{{ mariadb_sst_password }}"
     priv: "*.*:RELOAD,PROCESS,LOCK TABLES,BINLOG MONITOR"
 
+# This option defines the wsrep_sst_method that should be used by the cluster.
+# Possible options:
+#  - mariabackup - recommended for MariaDB - default if TLS is enabled via galera_sst_tls_enabled,
+#  - xtrabackup-v2 - there are limitations, please see https://mariadb.com/kb/en/xtrabackup-v2-sst-method/
```

Comment:
we should not list xtrabackup-v2, because install of respective packages is not handled yet

Reply:
Or we could just say it's unsupported by the role!

Reply:
Or we could just say it's unsupported by the role!